script.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable prefer-const */
+/* eslint-disable radix */
+/* eslint-disable max-classes-per-file */
```

Comment:
Kindly remove these disabled linters, linting improves readability, removes errors. 👍

Please check for other occurrences 🙂

Reply:
I made the changes, I only disabled the two that are mention in the project instructions:

TIP: If you have issues with the linter for either having multiple classes at the same file, or not an unused-variable error if you separated the code in multiple files, disable the linter check for that rule in the file with a /* eslint-disable rule-you-want-to-disable */ .